bundles/org.openhab.core.io.rest.auth/src/main/java/org/openhab/core/io/rest/auth/internal/AuthFilter.java

```diff
@@ -295,7 +294,9 @@ private List<CIDR> parseTrustedNetworks(String value) {
         var cidrList = new ArrayList<CIDR>();
         for (var cidrString : value.split(",")) {
             try {
-                cidrList.add(new CIDR(cidrString.trim()));
+                if (!value.isBlank()) {
```

Comment:
Shouldn't this `if` be moved outside of the `for` loop?

Reply:
It depends. If it is moved outside, something like `192.168.0.0/24,,192.168.1.0/24` (empty string in the middle) will generate a warning, if it is like it is now, this will be prevented, but we have more checks. OTOH, since this is only done when parsing the config, I believe it's ok as it is now.

Reply:
I would be able to follow your argument, if the if condition would be
```suggestion
                if (! cidrString.isBlank()) {
```

Reply:
Yes…